Language/hu/HTTP.php

```diff
@@ -27,7 +27,7 @@
     'emptySupportedNegotiations' => 'Minden kapcsolattípushoz tömbként szükséges megadni a támogatottakat.',
 
     // RedirectResponse
-    'invalidRoute' => '{0} útvonal nem található visszairányításkor.',
+    'invalidRoute' => '"{0}" útvonal nem található.',
```

Comment:
@kzolee1 can you confirm this change?

Reply:
@kzolee1 no longer available, I will continue to translate his work.

Reply:
Yes, I can confirm, the translation is correct.